### ISSUE TITLE
PRS on UK Biobank data

### ISSUE THREAD

**Issue by jiehuang001** [choishingwan/PRS-Tutorial#4]
Hi, It would be very nice to add a guide on LDpred-funct, and SbayesR, which are popular as well.

Also, it would be good to show how to generate PRS for UKB data (500,000 samples) with millions of SNPs. This seems to be a trend now.

Best regards,
Jie

**Comment by choishingwan** [comment]
I agree with you. Unfortunately, I don't currently have the time required
for me to code those up (and I might also need to update LDpred tutorial to
LDpred2) so that is unlikely to happen anytime soon. You are more than
welcome to help us write up a tutorial for those tools though. (don't think
codes for the latest LDpred-funct is available yet)

As for UK Biobank data, at least for PRSice and lassosum, you should be
able to run those scripts on the UK Biobank data just fine. Even the QC
steps should more or less translate to UK biobank data (with the exception
of maybe relatedness calculation). If you want scripts on UK Biobank QC,
you can find it here <https://gitlab.com/choishingwan/uk-biobank-piplines>

Regards,

Sam
